packages/react-composites/src/composites/CallWithChatComposite/adapter/AzureCommunicationCallWithChatAdapter.ts

```diff
@@ -394,6 +398,13 @@ export class AzureCommunicationCallWithChatAdapter implements CallWithChatAdapte
   public updateFileUploadMetadata = (id: string, metadata: FileMetadata): void => {
     this.chatAdapter.updateFileUploadMetadata(id, metadata);
   };
+  /* @conditional-compile-remove(teams-inline-images) */
+  async downloadAuthenticatedAttachment(attachmentUrl: string): Promise<AttachmentDownloadResult | undefined> {
+    if (this.chatAdapter.downloadAuthenticatedAttachment === undefined) {
```

Comment:
Why would this api be undefined? I think it is required by chatAdapter type?

Reply:
Looking at some of the comments above I think we will remove the undefined and we will just throw an error instead, like what James recommended and say something like `method not implemented`

Reply:
If we are throwing error like this, why don't we just mark the download method required in adapter type, it's actually required (because we are throwing errors but we just pretend it is not)

Again, overriding/creating own adapter is not the scenario we support for "non-breaking changes", cause it is not ACS related anymore